jupyter: python3
execute:
  daemon: false
  warning: false
